src/AnnotationDialog.js

```diff
@@ -241,15 +249,8 @@ class AnnotationDialog extends EventEmitter {
      * @return {void}
      */
     removeAnnotation(annotationID) {
-        const annotationEl = this.element.querySelector(`[data-annotation-id="${annotationID}"]`);
-        if (annotationEl) {
-            annotationEl.parentNode.removeChild(annotationEl);
-        }
-
-        const replyTextEl = this.element.querySelector(`.${CLASS_REPLY_TEXTAREA}`);
-        if (replyTextEl) {
-            replyTextEl.focus();
-        }
+        this.annotations = this.annotations.filter((annotation) => annotation.annotationID !== annotationID);
```

Comment:
you can unpack the annotation ID in the filter param:
`removeAnnotation(annotationIDToRemove) {`
...
`this.annotations.filter(({ annotationID }) => annotationID !== annotationIDToRemove);`